index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute_page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <h3>The man who saved a billion lives</h3>
+        </div>
```

Comment:
- It would be better if you use a header tag here instead of div as there should be more use of semantic tags for best HTML practices.

Reply:
Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change.